js/foundation.reveal.js

```diff
@@ -85,7 +85,7 @@ class Reveal {
   _makeOverlay(id) {
     var $overlay = $('<div></div>')
                     .addClass('reveal-overlay')
-                    .appendTo('body');
+                    .appendTo('this.options.appendTo');
```

Comment:
I agree, putting it in the quotes won't work, because jquery will try to find that string, rather than the variable.